### DISCUSSION TITLE
como trabajar con demas personas de manera remota y en mismos horarios

### DISCUSSION THREAD

**Discussion by JesusDiazTl** [community/community#66543] [New to GitHub]
hola, soy nuevo en la plataforma y quisiera que me orientaran sobre como puedo trabajar con mas usuarios y que la información del proyecto no se como trabajar varios remotamente y aparte que aguarde la información de todos sin que se dañe la de los demás

**Comment by queenofcorgis** [comment]
Hi @JesusDiazTl , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
